### DISCUSSION TITLE
See installed version and available update?

### DISCUSSION THREAD

**Discussion by silcotm** [plausible/analytics#1580] [Self-Hosted Support] [answered]
Hello,

Is there somewhere in the UI where I can see what version is installed?

Also, how do I know if an update becomes available?

**Comment by ukutaht** [comment, marked as answer]
This would be a great addition.

Unfortunately at the moment, you have to keep an eye on the Github releases to spot new ones. I'm sure there's a way to get a notification from Github when a new release is published

**Reply by networkpotato** [reply]
The documentation states

> The self-hosted version is somewhat of a LTS, only getting the changes after they have been battle tested on the hosted version. If you want features as soon as they are available, consider becoming a hosted customer.

Are the GitHub releases for the self-hosted version? Meaning, if a new release is posted to GitHub - will I get that new version if I pull the latest image? :)

**Reply by ukutaht** [reply]
Yes - Github Releases are the source of truth. When I create a new release I also upload the corresponding docker tags
